README.md

```diff
@@ -16,3 +16,34 @@ Command                                         | Description
 ```$ ./scripts/feeds/install device-manager```  | Install only the package device-manager
 
 
+### Building CreatorKit packages using pre-compiled OpenWrt SDK for Ci40 (Marduk):
+
+1. First download the latest SDK from https://downloads.creatordev.io/pistachio/marduk/ e.g. say  OpenWrt-SDK-0.9.4-pistachio-marduk_gcc-5.2.0_musl-1.1.11.Linux-x86_64.tar.bz2 is the latest SDK available.
+2. Extract the SDK and go to the extracted folder.
+
+          tar -xjvf OpenWrt-SDK-0.9.4-pistachio-marduk_gcc-5.2.0_musl-1.1.11.Linux-x86_64.tar.bz2
+          cd OpenWrt-SDK-0.9.4-pistachio-marduk_gcc-5.2.0_musl-1.1.11.Linux-x86_64.tar.bz2
+3. Check whether the feeds.conf.default file has an entry of https://github.com/CreatorKit/openwrt-ckt-feeds.git
+4. Update and install the openwrt-ckt-feeds by 
+
+          ./scripts/feeds update ckt
+          ./scripts/feeds install ckt
```

Comment:
This should be ./script/feeds update -a; ./scripts/feeds/install button-led-controller